test/rest_api/test_routing.clj

```diff
@@ -31,15 +31,15 @@
       (is (= (:body response) expected))))
   (testing
     "Main request handler produces correct data structure (fields)"
-    (let [req {:uri "/rest/field/gene/WBGene00000001/xrefs"
+    (let [req {:uri "/rest/field/gene/WBGene00000001/rearrangements"
```

Comment:
This file should be reverted to use `/xrefs`, with the other changes suggested here, it'll pass again.